webpack/ForemanTasks/Components/TasksDashboard/TasksDashboard.scss

```diff
@@ -1,6 +1,27 @@
+
```

Comment:
nitpik: can you please remove this empty line and add a new empty line at the end of the file?